src/client/components/Signup.tsx

```diff
@@ -70,19 +70,19 @@ export const SignUp: React.FC<RouteComponentProps> = ({history}) => {
                         </Tabs>
                     </Grid>
                     <Grid xs={12} item>
-                        <TextField label="Name" autoFocus value={name} onChange={onTextFieldUpdate(setName)}></TextField>
+                        <TextField id = "Name" label="Name" autoFocus value={name} onChange={onTextFieldUpdate(setName)}></TextField>
```

Comment:
Please use good naming practice for the id of each element: for id's please only use lowercase letters and words separated by "-" hyphens. Also please remove the spaces around the "=". Do this for all id fields in this file. Don't forget to update the test file with the new id's too

https://stackoverflow.com/questions/880776/best-practice-for-naming-the-id-attribute-of-dom-elements

Reply:
done